packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleTextViewer/SingleTextViewer.js

```diff
@@ -15,15 +15,21 @@ The `height` is defined in the ImageAndTextViewerContainer as the clientHeight p
 
 function SingleTextViewer ({
   content = '',
-  height = ''
+  height = '',
+  subjectId = ''
 }) {
   return (
     <Box
+      a11yTitle={`Subject ${subjectId} text`}
       flex='grow'
       height={{ min: height }}
       pad='xsmall'
+      role='document'
```

Comment:
Highlighting the addition of `a11yTitle`, `role`, and `tabIndex` here. 
I noticed the image viewer has an `aria-label` and is focusable, while the text viewer was not, so made the noted changes, but looking for feedback.

Reply:
`as=‘section’` might be better here, if you want to label a section of the page. 

All the landmark elements can be labelled. 
https://www.tpgi.com/short-note-on-aria-label-aria-labelledby-and-aria-describedby/

Reply:
Yes, great suggestion! Changed accordingly.